server/controllers/rooms.js

```diff
@@ -56,6 +56,24 @@ router.post('/join', auth, (req, res) => {
     }
 });
 
+/* Add an automated player to the room */
+router.post('/addAutoPlayer', (req, res) => {
+    try {
+        const { roomId }= req.body;
+        const x = Room.getById(roomId).gameState.getNumberofAutoPlayers();
```

Comment:
Rename `x` to be more descriptive? e.g. numAutoPlayers? Also can `getNumberofAutoPlayers();` have a capital O for Of